tools/backend/lsq-generator-python/vhdl_gen/cli.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+from vhdl_gen.configs   import GetConfigs
+from vhdl_gen.codegen   import codeGen
+
+def parse_args():    
+    # Parse the arguments
+    # python3 main.py [-h] [--target-dir PATH_RTL] --spec-file PATH_CONFIGS
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--target-dir', '-t', dest='path_rtl', default = './', type = str)
+    parser.add_argument('--spec-file', '-s', required = True, dest='path_configs', default = '', type = str)
+    args = parser.parse_args()
+
+    return args
+
+def main():
+    args = parse_args()
+    lsqConfig = GetConfigs(args.path_configs)
+    codeGen(args.path_rtl, lsqConfig)

```

Comment:
```suggestion
    code_gen.codeGen(args.path_rtl, lsqConfig)
```

this is super minor here but I like importing libraries as variables so it is clear where all your code comes from, directly importing functions/classes is slightly harder to read (maybe here the file is so small it is a bit pointless, but it is nice to be consistant)

Reply:
you can also rename the functions to be more generic (e.g. maybe the function from code gen is just called "generate" and the function from configs is just called "get") since you know they will be namespaced like this when called

Reply:
Understood. This will also be in the next PR.